contact.html

```diff
@@ -65,17 +66,17 @@
               <label for="comments">Tell us about your project *</label>
              </td>
              <td valign="top">
-              <textarea  name="comments" maxlength="1000" cols="25" rows="6"></textarea>
+              <textarea  name="comments" maxlength="1000" cols="25" rows="6" required></textarea>
              </td>
             </tr>
-
             <tr>
-             <td colspan="2" style="text-align:center">
-              <input type="submit" value="Submit"></a>
-             </td>
+              <td></td>
+              <td><input type="submit" value="Submit"></a></td>
             </tr>
           </table>
         </form>
+        <br>
+        <p id="alert-msg" class="alert-msg">Great to hear! We'll contact you back soon!</p>
```

Comment:
Please change message to 'Thank you for your inquiry. We will be in touch within the next business day!'